extensions/amp-analytics/0.1/config.js

```diff
@@ -63,12 +65,62 @@ export class AnalyticsConfig {
   loadConfig() {
     this.win_ = this.element_.ownerDocument.defaultView;
     this.isSandbox_ = this.element_.hasAttribute('sandbox');
+    // eslint-disable-next-line no-undef
+    const fetchVendorConfigPromise = ANALYTICS_VENDOR_SPLIT
+      ? this.fetchVendorConfig_()
+      : Promise.resolve();
 
-    return this.fetchRemoteConfig_()
+    return Promise.all([this.fetchRemoteConfig_(), fetchVendorConfigPromise])
       .then(this.processConfigs_.bind(this))
       .then(() => this.config_);
   }
 
+  /**
+   * Constructs the URL where the given vendor config is located
+   * @private
+   * @param {string} vendor the vendor name
+   * @return {string} the URL to request the vendor config file from
+   */
+  getVendorUrl_(vendor) {
+    const rtv = getMode().rtvVersion;
+    const baseUrl = getMode().localDev ? `/dist` : `${urls.cdn}/rtv/${rtv}`;
```

Comment:
Can we use the `#calculateScriptBaseUrl()` helper method?

Reply:
done :)